testsuite/integration/basic/pom.xml

```diff
@@ -2873,371 +3255,283 @@
-                                    <goal>jar</goal>
+                                    <goal>test</goal>
                                 </goals>
-                                <phase>process-test-classes</phase>
                                 <configuration>
-                                    <outputDirectory>${project.build.directory}/extraContent/modules/test/delta/main</outputDirectory>
-                                    <classesDirectory>${project.build.testOutputDirectory}</classesDirectory>
-                                    <includes>
-                                        <include>**/org/jboss/as/test/integration/weld/modules/DeltaBean.class</include>
-                                        <include>**/org/jboss/as/test/integration/weld/modules/DeltaLookup.class</include>
-                                        <include>**/META-INF/beans.xml</include>
-                                    </includes>
-                                    <finalName>delta</finalName>
-                                    <classifier>bootableJar</classifier>
+                                    <systemPropertyVariables>
+                                        <install.dir>${project.build.directory}/wildfly</install.dir>
+                                        <bootable.jar>${project.build.directory}/test-wildfly-cloud-profile.jar</bootable.jar>
+                                        <arquillian.xml>arquillian-bootable.xml</arquillian.xml>
+                                    </systemPropertyVariables>
+                                    <redirectTestOutputToFile>${testLogToFile}</redirectTestOutputToFile>
+                                    <classpathDependencyExcludes>
+                                        <classpathDependencyExclude>
+                                            org.wildfly.arquillian:wildfly-arquillian-container-managed
+                                        </classpathDependencyExclude>
+                                    </classpathDependencyExcludes>
+                                    <excludes combine.children="append">
+                                        <!-- For bootable-jar, custom modules have to be prepared aside in the time of jar packinging,
+                                        which is not sustainable for all tests. -->
+                                        <exclude>org/jboss/as/test/integration/security/vault/ExternalPasswordCommandsTestCase.java</exclude>
+                                        <exclude>org/jboss/as/test/integration/security/vault/ExternalPasswordByClassTestCase.java</exclude>
+                                        <exclude>org/jboss/as/test/integration/weld/modules/deployment/ModuleBeanDiscoveryModeTest.java</exclude>
+                                        <exclude>org/jboss/as/test/integration/weld/modules/export/ExportedModuleTestCase.java</exclude>
+                                        <!-- Requires extra modules -->
```

Comment:
these 3 excludes can be removed as the tests have been moved to testsuite/integration/microprofile

Reply:
Ok, done.